charts/ndb-operator/Chart.yaml

```diff
@@ -21,9 +21,15 @@ icon: https://www.nutanix.com/content/dam/nutanix/global/icons/products/svg/Nuta
 annotations:
   artifacthub.io/changes: |
     - kind: added
-      description: "Ability to provide Time Machine / SLA information"
-    - kind: fixed
-      description: "Error handling and minor bug fixes"
+      description: "Added the K8s Webhooks support for the validation of database specs"
```

Comment:
I have updated this part based on my understanding of the newer commits in the repo that were included in this PR. If you want to add/remove/update any content - do reach out to me separately.